metricbeat/module/docker/glide.yaml

```diff
@@ -2,3 +2,6 @@ package: github.com/elastic/beats/metricbeat/module/docker
 import:
 - package: github.com/fsouza/go-dockerclient
   version: 23c589186c2a92a8742da55f19aec4997dae5cbb
+import:
```

Comment:
I'm working here (https://github.com/elastic/beats/pull/3398) on updating the client. It is trickier then I thought. But as soon as this is merged you can rebase on top and you should not have to think about the vendor update part.